docs/restate/deployment.md

```diff
@@ -4,9 +4,11 @@ description: "Deploy Restate on Kubernetes with this guide."
 ---
 
 # Deployment
+
 Restate is currently a single binary that contains everything you need. You can obtain the binary from the
 [releases page](https://github.com/restatedev/restate/releases), as part of a
-[Docker image](https://hub.docker.com/r/restatedev/restate), from our [Homebrew tap](https://github.com/restatedev/homebrew-tap)
+[Docker image](https://hub.docker.com/r/restatedev/restate), from
+our [Homebrew tap](https://github.com/restatedev/homebrew-tap)
```

Comment:
Should we just link to here: https://restate.dev/get-restate/ ?

Reply:
Oh, yes! That would be a much better destination to link – thank you!

Reply:
This is better, I think: https://github.com/restatedev/documentation/pull/260/commits/e45a4702ec40f15f72594d068bdb7c1e2918c341